src/queryResult/queryResultWebViewController.ts

```diff
@@ -80,13 +84,22 @@ export class QueryResultWebviewController extends ReactWebviewViewController<
 
     private registerRpcHandlers() {
         this.registerRequestHandler("getRows", async (message) => {
-            return await this._sqlOutputContentProvider.rowRequestHandler(
+            let result = await this._sqlOutputContentProvider.rowRequestHandler(
                 message.uri,
                 message.batchId,
                 message.resultId,
                 message.rowStart,
                 message.numberOfRows,
             );
+            let currentState = this.getQueryResultState(message.uri);
+            if (currentState.isExecutionPlan) {
+                currentState.executionPlanState.xmlPlans =
+                    currentState.executionPlanState.xmlPlans.concat(
+                        result.rows[0][0].displayValue,
```

Comment:
add a comment: what should we expect in the first cell?  First column's name? Plan name?